docs/containers.conf.5.md

```diff
@@ -0,0 +1,347 @@
+empty or no valid path was found, then the `$PATH` environment variable will be
+used as the fallback.
+
+    The default list is:
+```
+conmon_path=[
+		"/usr/libexec/podman/conmon",
+		"/usr/local/libexec/podman/conmon",
+		"/usr/local/lib/podman/conmon",
+		"/usr/bin/conmon",
+		"/usr/sbin/conmon",
+		"/usr/local/bin/conmon",
+		"/usr/local/sbin/conmon",
+		"/run/current-system/sw/bin/conmon",
+]
+```
+
+**detach_keys**="ctrl-p,ctrl-q"
+  Keys sequence used for detaching a container.
+  Specify the keys sequence used to detach a container.
+Format is a single character `[a-Z]` or a comma separated sequence of
+`ctrl-<value>`, where `<value>` is one of:
+`a-z`, `@`, `^`, `[`, `\`, `]`, `^` or `_`
+
+**enable_port_reservation**=true
+  Determines whether libpod will reserve ports on the host when they are
+forwarded to containers. When enabled, when ports are forwarded to containers,
+they are held open by conmon as long as the container is running, ensuring that
+they cannot be reused by other programs on the host. However, this can cause
+significant memory usage if a container has many ports forwarded to it.
+Disabling this can save memory.
+
+**events_logger**="file"
+  Default method to use when logging events.
+  Valid values: `file`, `journald`, and `none`.
+
+**image_default_transport**="docker://"
+  Default transport method for pulling and pushing images.
+
+**infra_command**="/pause"
+  Command to run the infra container.
+
+**infra_image** = "k8s.gcr.io/pause:3.1"
+  Infra (pause) container image name for pod infra containers.  When running a
+pod, we start a `pause` process in a container to hold open the namespaces
+associated with the  pod.  This container and process, basically sleep/pause
+for the lifetime of the pod.
+
+**lock_type**="shm"
+  Specify the locking mechanism to use; valid values are "shm" and "file".
+Change the default only if you are sure of what you are doing, in general
+"file" is useful only on platforms where cgo is not available for using the
+faster "shm" lock type.  You may need to run "podman system renumber" after you
+change the lock type.
+
+**namespace**=""
+  Default libpod namespace. If libpod is joined to a namespace, it will see
+only containers and pods that were created in the same namespace, and will
+create new containers and pods in that namespace.  The default namespace is "",
+ which corresponds to no namespace. When no namespace is set, all containers
+and pods are visible.
+
+**no_pivot_root**=false
+  Whether to use chroot instead of pivot_root in the runtime.
+
+**num_locks**=2048
+  Number of locks available for containers and pods. Each created container or
+pod consumes one lock.  The default number available is 2048.  If this is
+changed, a lock renumbering must be performed, using the
+`podman system renumber` command.
+
+**runtime**=""
+  Default OCI runtime to use if nothing is specified in **runtimes**.
```

Comment:
This is the specific runtime in `runtimes` that will be used by default. Must refer to a member of the `runtimes` table.

Reply:
Fixed